_posts/2019-07-09-javelin.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Javelin project
+---
+
+We are moving gfx-rs project forward, while having fun with it, thanks to the power provided by Rust language. We are building rich abstractions, which expand beyon gfx-hal API itself - into the internal layers of the backends, structured to be modular and maintainable. We are building high-performance graphics and compute infrastructure that deeply interacts with OS and drivers, thanks to the Rust's FFI capabilities and the lack of runtime. All of that, while dedicating only a bit of time and attention from the developers, who mostly do it as a side project. Rust allows us to move forward confidently and quickly, experiment with features as well as land the production code.
```

Comment:
We are having fun moving the gfx-rs project forward thanks to the power provided by the Rust language.

We are building rich abstractions which expand beyond the gfx-hal API itself and into the internal layers of the backends, structured to be modular and maintainable.

We do all of this while requiring only a portion of our developers' time and attention, who work on gfx-rs mostly as a side project.

Reply:
do you think the other sentences there are best to be removed?

Reply:
No, just only had comments on those ones :)